Ollivanders/src/net/pottercraft/Ollivanders2/Potion/MEMORY_POTION.java

```diff
@@ -10,12 +15,28 @@
  */
 public final class MEMORY_POTION extends Potion
 {
-   public MEMORY_POTION ()
+   public MEMORY_POTION (Ollivanders2 plugin)
    {
+      super(plugin);
+
+      name = "Memory Potion";
```

Comment:
Is this a duplicate line?

Reply:
Looks like a merge conflict that I missed.